packages/validator/src/services/block.ts

```diff
@@ -131,6 +141,7 @@ export class BlockProposingService {
         feeRecipient,
         strictFeeRecipientCheck,
         builderSelection,
+        blindedLocal,
```

Comment:
need to enhance `produceBlockWrapper` and `produceBlockV2Wrapper` too?

Reply:
no this only addresses v3 api, blinded flow can otherwise happen only througn produceBlindedBlock which anyway returns blinded no matter local or builder selected